<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmManShs</class>
 <widget class="QMainWindow" name="frmManShs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHS MANAGER</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>9</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 24pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>SHS MANAGER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>4</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/icons/1426825-sales/png/001-shop.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbTag_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>48</y>
      <width>571</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>TAG DETAILS</string>
    </property>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>24</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* MACHINE CODE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMachCode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>26</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMachName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>48</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>46</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* MACHINE NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSerial">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>68</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* SERIAL NO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>93</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>91</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>STATUS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSimNo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>26</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>24</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>SIM NO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>46</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>FIRMWARE VER.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFirmware">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>48</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSoftware">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>293</x>
       <y>68</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>SOFTWARE VER.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPkeyMach">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>95</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbTag_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>170</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>CUSTOMER TAG</string>
    </property>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>47</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>CUSTOMER NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCusName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>49</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>22</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>CODE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCusCode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>27</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboCustomer">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>72</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>67</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>CUSTOMER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>269</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="txtCusNo_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>25</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* SHS NO.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>25</y>
       <width>92</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbMachList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>551</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Record(s) Found</string>
     </property>
     <widget class="QTableWidget" name="tblMachList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>CODE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MACH NAME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SERIAL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SIM</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FIRMWARE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SOFTWARE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CUSTOMER CODE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CUSTOMER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MACHINE NO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>547</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/010-close.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/010-close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>547</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/004-disabled.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/004-disabled.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>546</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/092-notes.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/092-notes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>546</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/090-plus.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/090-plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>547</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/062-diskette.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/062-diskette.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>frmManShs</receiver>
   <slot>btnAddShsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdit</sender>
   <signal>clicked()</signal>
   <receiver>frmManShs</receiver>
   <slot>btnEditShsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>frmManShs</receiver>
   <slot>btnDeleteShsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>frmManShs</receiver>
   <slot>btnSaveShsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>frmManShs</receiver>
   <slot>btnCancelShsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblMachList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>frmManShs</receiver>
   <slot>fetchTblRowDataShs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnAddShsClicked()</slot>
  <slot>btnEditShsClicked()</slot>
  <slot>btnDeleteShsClicked()</slot>
  <slot>btnSaveShsClicked()</slot>
  <slot>btnCancelShsClicked()</slot>
  <slot>fetchTblRowDataShs()</slot>
 </slots>
</ui>
